app/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -18,6 +18,11 @@
       document.querySelector('html').style.setProperty("--item-size", size + 'px');
     });
 
+    $scope.$watch('app.settings.showNewAnimation', function(showAnimation) {
+      var animation = showAnimation ? 'new_overlay 5s infinite' : 'initial';
+      document.querySelector('html').style.setProperty("--overlay-animation", animation);
```

Comment:
Rather than doing this, just set a class on body, and in the CSS, only turn on the animation when the selector matches. For example, if the class you add is called `new-item-animated`, the SCSS could be:

```scss
.new_overlay {
  width: var(--item-size);
  height: var(--item-size);
  pointer-events: none;
  .new-item-animated & {
    will-change: transform;
    animation: new_overlay 4s linear infinite;
  }
}
```